docs/admin/setup/pyris.rst

```diff
@@ -1,100 +1,332 @@
-
-Start Pyris
-"""""""""""
-
-You can now start Pyris using the following command:
-
-.. code-block:: bash
-
-   docker compose --project-directory "$PROJECT_DIR" -f "$COMPOSE_FILE" --env-file "$ENV_FILE" up -d --pull always --no-build
-
-- ``$PROJECT_DIR`` should point to the Pyris directory you created.
-- ``$COMPOSE_FILE`` should point to the ``docker/pyris-production.yml`` file in the Pyris repository.
-- ``$ENV_FILE`` should point to the ``.env`` file you just created.
-
-E.g.:
-
-.. code-block:: bash
-
-   docker compose --project-directory "/opt/pyris" -f "/opt/pyris/Pyris/docker/pyris-production.yml" --env-file "/opt/pyris/.env" up -d --pull always --no-build
-
-This will start the Pyris application on your server/VM.
+          - token: "your-secret-token"
+
+        weaviate:
+          host: "localhost"
+          port: "8001"
+          grpc_port: "50051"
+
+        env_vars:
+          test: "test-value"
+
+   - **Create LLM Config File**
+
+     Create an ``llm_config.local.yml`` file in the root directory. This file includes a list of models with their configurations.
+
+     Example command:
+
+     .. code-block:: bash
+
+        cp Pyris/llm_config.example.yml llm_config.local.yml
+
+     **Example OpenAI Configuration**
+
+     .. code-block:: yaml
+
+        - id: "oai-gpt-35-turbo"
+          name: "GPT 3.5 Turbo"
+          description: "GPT 3.5 16k"
+          type: "openai_chat"
+          model: "gpt-3.5-turbo"
+          api_key: "<your_openai_api_key>"
+          tools: []
+          capabilities:
+            input_cost: 0.5
+            output_cost: 1.5
+            gpt_version_equivalent: 3.5
+            context_length: 16385
+            vendor: "OpenAI"
+            privacy_compliance: false
+            self_hosted: false
+            image_recognition: false
+            json_mode: true
+
+     **Example Azure OpenAI Configuration**
+
+     .. code-block:: yaml
+
+        - id: "azure-gpt-4-omni"
+          name: "GPT 4 Omni"
+          description: "GPT 4 Omni on Azure"
+          type: "azure_chat"
+          endpoint: "<your_azure_model_endpoint>"
+          api_version: "2024-02-15-preview"
+          azure_deployment: "gpt4o"
+          model: "gpt4o"
+          api_key: "<your_azure_api_key>"
+          tools: []
+          capabilities:
+            input_cost: 6
+            output_cost: 16
+            gpt_version_equivalent: 4.5  # Equivalent GPT version of the model
+            context_length: 128000
+            vendor: "OpenAI"
+            privacy_compliance: true
+            self_hosted: false
+            image_recognition: true
+            json_mode: true
+
+     **Explanation of Configuration Parameters**
+
+     The configuration parameters are utilized through the capability system by pipelines in Pyris to select the appropriate model for a task.
+
+     **Parameter Descriptions:**
```

Comment:
I've marked some parameters above as they need more details here. Especially, implications are important for admins.

Reply:
Thanks for the review. I have added some additional text to highlight the implications of the values set for specific parameters.